AUTHORS

```diff
@@ -1266,4 +1268,50 @@ Augusto Borges <borges.augustoar@gmail.com>
 Han Wei Ang <ang.h.w@u.nus.edu>
 Pablo <48098178+PabloRuizCuevas@users.noreply.github.com>
 Congxu Yang <u7189828@anu.edu.au>
-Saicharan <62512681+saicharan2804@users.noreply.github.com>
\ No newline at end of file
+Saicharan <62512681+saicharan2804@users.noreply.github.com>
+Arnab Nandi <arnabnandi2002@gmail.com>
+Harrison Oates <48871176+HarrisonOates@users.noreply.github.com>
+Corey Cerovsek <corey@cerovsek.com>
+Harsh Kasat <hkasat@waymore.io>
+omahs <73983677+omahs@users.noreply.github.com>
```

Comment:
gh-25972

@omahs is this how you want your name to be recorded in the AUTHORS file?

Usually we put contributor's real name and email address in here.

Reply:
Yes, all good with **omahs**, thanks!

Reply:
Okay.